tests/e2e/General/UsageTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Scopes\SideServer;
+
+class UsageTest extends Scope
+{
+    use ProjectCustom;
+    use SideServer;
+
+    protected array $headers = [];
+    protected string $projectId;
+
+    protected int $usersCount = 0;
+    protected int $requestsCount = 0;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $project = $this->getProject(true);
+        $this->projectId = $project['$id'];
+        $this->headers['x-appwrite-project'] = $project['$id'];
+        $this->headers['x-appwrite-key'] = $project['apiKey'];
+        $this->headers['content-type'] = 'application/json';
+    }
+
+    public function testUsersStats(): void
+    {
+
+        for ($i = 0; $i < 10; $i++) {
+            $email = uniqid() . 'user@usage.test';
+            $password = 'password';
+            $name = uniqid() . 'User';
+            $res = $this->client->call(Client::METHOD_POST, '/users', $this->headers, [
+                'userId' => 'unique()',
+                'email' => $email,
+                'password' => $password,
+                'name' => $name,
+            ]);
+            $this->assertEquals($email, $res['body']['email']);
+            $this->assertNotEmpty($res['body']['$id']);
+            $this->usersCount++;
+            $this->requestsCount++;
+        }
+        sleep(65);
```

Comment:
In order to compensate for stuff like this, we really need to move some Functions Test over to open-runtimes @christyjacob4

Reply:
I think we can move most of them, as long as one function works on Appwrite we should be fine. Open Runtimes should provide most of the coverage.